cloudbuild.yaml

```diff
@@ -1,15 +1,15 @@
 options:
   substitution_option: ALLOW_LOOSE
 steps:
-  - name: gcr.io/k8s-testimages/gcb-docker-gcloud
+  - name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20211118-2f2d816b90
```

Comment:
@cpanato looks like newer ones are available? https://cs.github.com/?scopeName=All+repos&scope=&q=gcr.io%2Fk8s-staging-test-infra%2Fgcb-docker-gcloud%3Av2022+org%3Akubernetes 

can you please update?

Reply:
done, ptal
thanks for the catch